src/core/core/global_config.cpp

```diff
@@ -443,11 +468,36 @@ bool service_spec::init_app_specs()
         ))
         return false;
 
-    std::vector<std::string> allSectionNames;
-    config->get_all_sections(allSectionNames);
+    std::vector<std::string> all_section_names;
+    config->get_all_sections(all_section_names);
     
+    // check mimic app
+    if (enable_default_app_mimic)
+    {
+        std::string mimic_section_name("apps.mimic");
+        if (std::find(all_section_names.begin(), all_section_names.end(), mimic_section_name)
+            == all_section_names.end())
+        {
```

Comment:
done in https://github.com/imzhenyu/rDSN/commit/fe1aa9c4c31bc2350c038713bbf111a7bbef9db9